apiserver/facades/client/resources/server_listresources_test.go

```diff
@@ -1,18 +1,22 @@
 // Copyright 2017 Canonical Ltd.
```

Comment:
Whilst you're changing this file, can you change the name of it as well? It could just be called facade_test.go even.